Firebase/Auth/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# v5.1.0
```

Comment:
Usual practice is "Unreleased" section for changes this early - but probably fine since we know it will be 5.1.0.